tests/data_types/text/run.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Copyright (C) 2019-2020 Siodb GmbH. All rights reserved.
+# Use of this source code is governed by a license that can be found
+# in the LICENSE file.
+
+## Global
+SCRIPT_DIR=$(dirname $(realpath "$0"))
+TEST_NAME=$(basename "${SCRIPT_DIR}")
+source "${SCRIPT_DIR}/../../share/CommonFunctions.sh"
+
+## Specific test functions
+
+## Specific test parameters
+
+## =============================================
+## TEST HEADER
+## =============================================
+_TestBegin
+_Prepare
+_StartSiodb
+
+## =============================================
+## TEST
+## =============================================
+
+# TODO: Once issue https://github.com/siodb/siodb/issues/94 is fixed
+# output_dir="${HOME}/tmp/export_$(date +%s)"
```

Comment:
add `_$$` to directory name

Reply:
@cxxman did not understand this one. Have you an example?

Reply:
`"${HOME}/tmp/export_$(date +%s)_$$`
https://unix.stackexchange.com/questions/291570/what-is-in-bash